app/models/metadatum.rb

```diff
@@ -8,7 +8,7 @@ class Metadatum < ApplicationRecord
 
   if ELASTICSEARCH_ON
     include Elasticsearch::Model
-    include Elasticsearch::Model::Callbacks
+    include ElasticsearchCallbacksHelper
```

Comment:
I want to see this declared loudly as a comment whenever `ElasticsearchCallbacksHelper` is included.

>  Any modifications that circumvent the Active Record Callback Flow result in missing data in Elasticsearch.

That should help spread the knowledge in the team.